extension/content.js

```diff
@@ -328,6 +368,11 @@ document.addEventListener('DOMContentLoaded', () => {
 			if (isCommit()) {
 				addPatchDiffLinks();
 			}
+
+			if (isPRFiles()) {
```

Comment:
@DrewML Ultimately you'd use a check which will catch either `/pull/<id>/files` or `<pull/<id>/commits/<sha>`.